frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json

```
@@ -271,7 +271,7 @@
   "This is a required field. The Storage Class will be used to request storage from the underlying infrastructure to create the backing persistent volumes that will be used to provide the OpenShift Container Storage (OCS) service.": "This is a required field. The Storage Class will be used to request storage from the underlying infrastructure to create the backing persistent volumes that will be used to provide the OpenShift Container Storage (OCS) service.",
   "Create new storage class": "Create new storage class",
   "All required fields are not set": "All required fields are not set",
-  "In order to create the storage cluster you must set the storage class select at least 3 nodes (preferably in 3 different zones) and meet the minimum or recommended requirement": "In order to create the storage cluster you must set the storage class select at least 3 nodes (preferably in 3 different zones) and meet the minimum or recommended requirement",
+  "In order to create the storage cluster you must set the storage class, select at least 3 nodes (preferably in 3 different zones), and meet the minimum or recommended requirement": "In order to create the storage cluster you must set the storage class, select at least 3 nodes (preferably in 3 different zones), and meet the minimum or recommended requirement",
```

The second comma added here is not needed.

```suggestion
  "In order to create the storage cluster you must set the storage class, select at least 3 nodes (preferably in 3 different zones) and meet the minimum or recommended requirement": "In order to create the storage cluster you must set the storage class, select at least 3 nodes (preferably in 3 different zones) and meet the minimum or recommended requirement",
```

---

Should be all set!